cmd/puma-dev/command.go

```diff
@@ -15,11 +19,65 @@ func command() error {
 	switch flag.Arg(0) {
 	case "link":
 		return link()
+	case "status":
+		return status()
 	default:
 		return fmt.Errorf("Unknown command: %s\n", flag.Arg(0))
 	}
 }
 
+type App struct {
+	Status  string `json:"status"`
+	Scheme  string `json:"scheme"`
+	Address string `json:"address"`
+}
+
+func status() error {
+	client := &http.Client{}
+	req, err := http.NewRequest("GET", "http://localhost/status", nil)
+	req.Host = "puma-dev"
+	w := tabwriter.NewWriter(os.Stdout, 20, 4, 1, ' ', 0)
+
+	if err != nil {
+		return err
+	}
+
+	res, err := client.Do(req)
+
+	if err != nil {
+		fmt.Println("puma-dev is not running")
```

Comment:
i'm not sure if we have enough information to say for sure that puma dev isn't running. what we do know is that we weren't able to get the status from `GET localhost:80/status`.

by way of heading off future new issues saying "puma-dev is running but status says it's not!!1", how do you feel about `fmt.Sprintf("unable to lookup puma-dev status. is puma-dev listening on port %s?", fHTTPPort)` ?

Reply:
Totally cool with it.